lib/manageiq_performance/middlewares/active_record_queries.rb

```diff
@@ -153,12 +153,29 @@ def skip_schema_queries?
         IGNORED_PAYLOAD    = %w[EXPLAIN CACHE].freeze
 
         def ignore_payload?(payload)
-          payload[:exception] ||
+          payload[:exception] || payload[:cached] ||
```

Comment:
:+1:

Reply:
This is what I was actually curious about your opinion on, since this class is based on your code, even the comment ;)

So above, there is the `IGNORED_PAYLOAD`, which I couldn't find in the `ActiveRecord` codebase where `CACHE` ends up in the `payload[:name]`, but I did find `payload[:cached]` so included that in as well.  That said, I could also see there being some use of keeping track of these as well, since while they avoid the network call to the DB, pretty sure they still have to `.dup` the records before returning them when called.

This would probably still show up in the `'instantiation.active_record'` notifier, but still, it would be hard to find where this is coming from if that is the case.  Anyway, just something I was curious about your thoughts on.